tests/jballerina-unit-test/src/test/resources/test-src/types/function/function-type.bal

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Let's add a runtime test for `is function` too.

Reply:
Added.